frontend/public/components/utils/details-page.tsx

```diff
@@ -24,6 +24,8 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({children, reso
     <dd>{metadata.name || '-'}</dd>
     { metadata.namespace ? <dt>Namespace</dt> : null }
     { metadata.namespace ? <dd><ResourceLink kind="Namespace" name={metadata.namespace} title={metadata.uid} namespace={null} /></dd> : null }
+    { type ? <dt>Type</dt> : null }
+    { type ? <dd>{type}</dd> : null }
```

Comment:
@spadgett saw your comment on the PR in the bridge repo, but not sure if we want to show the `Type` field if the resource hasn't got the `type` field. Do we ?

Reply:
Either way seems fine. If we change our minds on this, it can be fixed in a subsequent PR.